cmake/external-libraries.cmake

```diff
@@ -41,6 +41,19 @@ if(KPHP_TESTS)
     endif()
 endif()
 
+find_library(KPHP_TIMELIB kphp-timelib)
```

Comment:
It would be better to add `kphp-timelib` to the variable  `CPACK_DEBIAN_KPHP_PACKAGE_DEPENDS`, so the deb package will depend on the library

Reply:
Done.